README.md

```diff
@@ -1,49 +1,65 @@
 TomQueue
 =========
 
-TomQueue hooks onto [delayed_job_active_record](https://github.com/collectiveidea/delayed_job_active_record) to move the work-load queue out of the database onto a more suitable queueing server, in our case [RabbitMQ](http://rabbitmq.com).
+TomQueue is a backend for [Delayed::Job](https://github.com/collectiveidea/delayed_job) gem. TomQueue hooks onto [delayed_job_active_record](https://github.com/collectiveidea/delayed_job_active_record) backend and replaces the mechanism by which Delayed::Job workers acquire jobs. By default Delayed::Job workers poll the database for new jobs. TomQueue replaces "polling" logic with subscription to [RabbitMQ](http://rabbitmq.com) queue. Delayed::Job workers receive a new job as soon as it gets published to the queue.
 
 Why?
 ----
 
-At FreeAgent, we have always used Delayed Job to manage asynchronous work and find it still fits our needs well, not only in getting code running in production but also in its handling of failed jobs as well as ensuring our entire engineering team understand how it behaves. It's a core part of our workflow and infrastructure and we're happy with it.
+At FreeAgent, we have always used Delayed::Job to manage asynchronous work and find it still fits our needs well. That said, when it is backed by MySQL, we've found that it performs particularly poorly when the work queued gets large (i.e. 10k+). In fact, the larger the queue of work gets, the *slower* the query to pull the next job! The more Delayed:::Job workers are running the bigger problem it becomes.
```

Comment:
> The more Delayed:::Job workers are running the bigger problem it becomes.

The more **Delayed::Job** workers **that** are running the bigger a problem it is.